.connector-store/meta.json

```diff
@@ -1,25 +1,20 @@
 {
-    "name": "SharePoint",
-    "owner": "WSO2",
-    "category": "Storage and Content Management",
-    "documentationUrl": "https://docs.wso2.com/display/ESBCONNECTORS/SharePoint+Connector",
-    "description": "The sharePoint connector allows you to access the SharePoint REST API through WSO2 ESB. SharePoint is a web application platform.SharePoint combines various functions which are traditionally separate applications: intranet, extranet, content management, document management, enterprise search, workflow management and web content management. ",
-    "status": "Inactive",
-    "labels": [
-        "storage"
-    ],
-    "releases": [
-        {
-            "tagName": "org.wso2.carbon.connector.sharepoint-1.0.5",
-            "products": [
-                "EI 6.4.0",
-                "EI 6.3.0",
-                "EI 6.2.0",
-                "EI 6.1.1",
-                "ESB 5.0.0",
-                "ESB 4.9.0"
-            ],
-            "isHidden": false
-        }
-    ]
-}
\ No newline at end of file
+  "name": "SharePoint",
+  "owner": "WSO2",
+  "category": "Storage and Content Management",
+  "documentationUrl": "https://mi.docs.wso2.com/en/latest/reference/connectors/sharepoint-connector/sharepoint-overview/",
+  "description": "The sharePoint connector allows you to access the SharePoint REST API. SharePoint is a web application platform.SharePoint combines various functions which are traditionally separate applications: intranet, extranet, content management, document management, enterprise search, workflow management and web content management.",
+  "status": "Inactive",
```

Comment:
Shall we make this Active?

Reply:
1d9ee46e29b3593c64ec9f9d87846511284dd2da